oada-core/http-handler/oada-srvc-http-handler/resources.js

```diff
@@ -18,6 +18,18 @@ const changes = require('../../libs/oada-lib-arangodb').changes;
 const putBodies = require('../../libs/oada-lib-arangodb').putBodies;
 const OADAError = require('oada-error').OADAError;
 
+const { pipeline } = require('stream');
+const pipelineAsync = require('bluebird').promisify(pipeline);
+const cacache = require('cacache');
+
+const multer = require('multer');
+const fileUpload = multer({ limits: '20mb' });
+// const Readable = require('stream').Readable;
+// const fs = require('fs');
+
+// check that path exists?
+const CACHE_PATH = "tmp/oada-cache";
```

Comment:
This should come from `oada-config`
```suggestion
const CACHE_PATH = config.get("storage:binary:cacache");
```